source/deployment/elasticsearch.rst

```diff
@@ -3,7 +3,7 @@ Elasticsearch (E20)
 
 *Available in Enterprise Edition E20.*
 
-The Elasticsearch module in Mattermost E20 is recommended for enterprise-scale deployments to optimize search performance and avoid performance degradation and timeouts.
+Elasticsearch provides enterprise-scale deployments optimized search performance and prevents performance degradation and timeouts. 
```

Comment:
```suggestion
Elasticsearch provides enterprise-scale deployments for optimized search performance and prevents performance degradation and timeouts.
```
?

Reply:
I just saw that and added "with". I think that works?

Reply:
Yes, thanks